crates/telio-nurse/src/aggregator.rs

```diff
@@ -322,11 +330,18 @@ impl<W: WireGuard> ConnectivityDataAggregator<W> {
         }
     }
 
+    #[allow(dead_code)] //TODO(Hasan) : Remove this!
```

Comment:
Adding a reminder github thread for TODO item ;)

Reply:
Have already created a jira item and assigned myself to it. :)

Reply:
Because there was a lot of code to be added that was kind of being duplicated (Being already added in https://github.com/NordSecurity/libtelio/pull/396)

Reply:
And well to avoid merge conflicts because there are some other changes as well made in https://github.com/NordSecurity/libtelio/pull/396 later on that I haven't made

Reply:
If I am not mistaken, this dead_code annotation is not needed anymore?